modules/conflict_resolver/README.md

```diff
@@ -66,3 +66,6 @@ If the `conflicts_unresolved` table somehow becomes out of sync
 with the data in LORIS, the `tools/recreate_conflicts.php` script
 can be used to re-compare single and double data entry values
 and repopulate the `conflicts_unresolved` table.
+
+The conflict resolver registers modules on the dashboard and the
+`candidate_profile` dashboard.
```

Comment:
```suggestion
`candidate_profile` modules.
```

I don't like calling it a dashboard to avoid confusion with THE dashboard which is a module

Reply:
It's emphatically designed to be a dashboard.